packages/frontend-shared/src/utils/spec-utils.ts

```diff
@@ -4,8 +4,11 @@ import { ComputedRef, Ref, ref, watch } from 'vue'
 import _ from 'lodash'
 
 import type { UseCollapsibleTreeNode } from '../composables/useCollapsibleTree'
+import { decodeBase64Unicode } from './decodeBase64'
 
-const PATH_SEP = /[\/\\]/
+const platform = JSON.parse(decodeBase64Unicode(window.__CYPRESS_CONFIG__.base64Config)).platform
+const regexSeparator = platform === 'win32' ? /\\/ : /\//
+const separator = platform === 'win32' ? '\\' : '/'
```

Comment:
Actually this entire file should not be in `frontend-shared`, it's only used in `app`. How about moving it?